charters/DataOperations/charter.md

```diff
@@ -0,0 +1,101 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team (DataOps) is a user-focused group aiming to connect the scientific community to the DCP through the effective tools and resources generated by the DCP which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution. These products include data, metadata, analysis tools, data access features. DataOps ensures that data flows through the DCP end-to-end in an efficient manner, is appropriately validated, and fulfills the needs and expectations of data consumers. DataOps maintains a DCP-wide vision through their internal focus on end-to-end data flow, as well as their community-focused goal of conveying the value of the entire DCP.
+
+## Definitions
+
+*Data contribution*: a set of raw data derived from the same experimental process, and the metadata describing that process
+
+*Release* (verb): to make metadata and data openly accessible
+
+*Data distribution*: the packaging of metadata and data from multiple data contributions in meaningful ways, along with relevant analysis pipelines, possible integrative analysis results, user vignettes, access features, and any announcements or documentation supporting it
+
+*HCA Snapshot*: a type of data distribution that occurs on a regular cadence containing all HCA data currently available that meets predetermined criteria, such as quality standards
+
+*Data workflow*: the sequence of tasks executed by the DCP for each data contribution beginning with first notification of a dataset (typically contact with the data generator or identification of data in a data archive or in scientific literature) all the way through data analysis by the DCP and continued through usability by data consumers
+
+## Objectives
+
+The objective of the Data Operations is to connect the scientific community to tools and resources from the DCP which maximize the value and use of HCA data such as data, metadata, analysis tools, and data access features. Data Operations aims to maintain a reliable and efficient data workflow and shepherds data through the data workflow.
```

Comment:
I like the first sentence, "connect" is key here and, in that context, this sentence comes together nicely. The second sentence cites this team as being responsible for data flow being "reliable and efficient". This is everyone's job. Also, I do not see explicit language here about curation, packaging, release, and distribution. If I misunderstand your team, please feel free to say so.

Reply:
Each team is responsible for data flow, but their scope is limited to within their component, and rightfully so.
DataOps has the end-to-end scope, adding tremendous value as a single data shepherd that deeply understands and supports all of thee other components. This allows us to own the gaps (hand-offs between components) and also gives us a good perspective to understand where components are misunderstanding or unaware of aspects of another component, where validation should occur, and how to make the _entire_ lifecycle as efficient as possible.
Curation, packaging, releasing, and distribution are all ways in which we will connect DCP-to-community, so that is covered in the first sentence (or at least, that's the intention).

Reply:
@TimothyTickle does this answer your questions, or is there still more to resolve here?

Reply:
Not blocking